--- ldc-1.42.0-beta1-src/runtime/CMakeLists.txt.orig	2025-12-19 13:54:08.000000000 +0100
+++ ldc-1.42.0-beta1-src/runtime/CMakeLists.txt	2026-02-01 14:15:09.737813213 +0100
@@ -276,7 +276,7 @@
 # Default -rpath linker option when linking against shared libraries.
 if(SHARED_LIBS_SUPPORTED)
     set(build_rpath   "${build_libdir}")
-    set(install_rpath "${install_libdir}")
+    set(install_rpath "")
 endif()
 
 # This is e.g. set from `ldc-build-runtime --installWithSuffix`.
